client/test/setup.go

```diff
@@ -62,3 +73,84 @@ func MakeRoleSetups(rng *rand.Rand, s *ethctest.Setup, names []string) []clientt
 	}
 	return setups
 }
+
+// MakeNetRoleSetups creates a two party client test setup with the provided names and uses the default TLS-bus from go-perun.
+func MakeNetRoleSetups(t *testing.T, rng *rand.Rand, s *ethctest.Setup, names []string) []clienttest.RoleSetup {
+	t.Helper()
+	setups := make([]clienttest.RoleSetup, len(names))
+	commonName := "127.0.0.1"
+	sans := []string{"127.0.0.1", "localhost"}
+	tlsConfigs, err := GenerateSelfSignedCertConfigs(commonName, sans, len(names))
+	if err != nil {
+		panic("Error generating TLS configs: " + err.Error())
+	}
+
+	hosts := make([]string, len(names))
+	for i := 0; i < len(names); i++ {
+		port, err := findFreePort()
```

Comment:
Auf lange Sicht wird es vermutlich besser sein eine Funktion zum listener hinzuzufügen (wenn er nicht schon eine hat) um den Port zu bekommen. Einen tcp listener aufzumachen nur um den Port zu bekommen erscheint mir unnötig.

Also effektiv erst die listener erdstellen und dann für jeden listener den Port abfragen (wie es in findFreePorts gemacht wird) + evtl. kleine Anpassung am perun code um diese Funktionalität bereitzustellen.